x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -16,15 +18,14 @@ interface CreateExecuteFunctionOptions {
   preconfiguredActions: PreConfiguredAction[];
 }
 
-export interface ExecuteOptions {
+export interface ExecuteOptions extends Pick<ActionExecutorOptions, 'params' | 'source'> {
   id: string;
-  params: Record<string, unknown>;
   spaceId: string;
   apiKey: string | null;
 }
 
 export type ExecutionEnqueuer = (
-  savedObjectsClient: SavedObjectsClientContract,
+  unsecuredSavedObjectsClient: SavedObjectsClientContract,
```

Comment:
This *was* unsecured before, but it wasn't named as such. As we've marked the other ones in this way, it felt like we should do that here too.